rmw_connextdds_common/src/common/rmw_node.cpp

```diff
@@ -229,3 +229,9 @@ rmw_api_connextdds_node_get_graph_guard_condition(const rmw_node_t * rmw_node)
 
   return node_impl->graph_guard_condition();
 }
+
+rmw_ret_t
+rmw_notify_participant_dynamic_network_interface(rmw_context_t * context)
```

Comment:
Minor comment, which applies more to the rmw interface that this specific change, but I always prefer to pass `const` pointers, e.g.:

```suggestion
rmw_notify_participant_dynamic_network_interface(rmw_context_t * const context)
```